lighttpd/tests/test_check.py

```diff
@@ -35,13 +41,21 @@ def test_lighttpd(aggregator, check, instance):
     aggregator.assert_all_metrics_covered()
 
 
-@pytest.mark.usefixtures('dd_environment')
-@pytest.mark.integration
 def test_service_check_ko(aggregator, check, instance):
-    """
-    """
     instance['lighttpd_status_url'] = 'http://localhost:1337'
     tags = ['host:localhost', 'port:1337', 'instance:first']
     with pytest.raises(Exception):
         check.check(instance)
     aggregator.assert_service_check(check.SERVICE_CHECK_NAME, status=Lighttpd.CRITICAL, tags=tags)
+
+
+@pytest.mark.e2e
+def test_check(aggregator):
```

Comment:
would a new tox environment be required?

Reply:
No it reuses existing environments.

Reply:
Ok, just noting down that my concern was about how we run e2e vs integrations and the answer is one uses the check command and the other the test command